crates/conductor_lib/src/conductor/admin.rs

```diff
@@ -903,13 +905,15 @@ id = 'new-dna'"#,
             vec![
                 DnaConfiguration {
                     id: String::from("test-dna"),
-                    file: String::from("app_spec.dna.json"),
+                    file: None,
```

Comment:
lets add a comment onto these `file: None` about the deprecation so it's clear in the code that they are going away.